recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -24,9 +24,9 @@
             :class="[
               'd-h24 d-bar16 d-py0 d-fs-200 d-mb0 d-t',
               !!reaction.isSelected
-                ? `d-fc-purple-500 d-bgc-purple-400 d-bgo10 d-bc-purple-400 d-bco50 ' +
-                  'h:d-bgc-purple-400 h:d-bco25 a:d-bgc-purple-400 a:d-bco25`
-                : 'd-fc-black-600 d-bgc-black-100 h:d-bgc-white h:d-bc-black-600 a:d-bgc-white a:d-bc-black-600',
+                ? `d-fc-purple-500 d-bgc-purple-400 d-bgo10 d-bc-purple-400 d-bco50 ` +
+                  `h:d-bgc-purple-400 h:d-bco25 f:d-bgc-purple-400 f:d-bco25`
```

Comment:
Maybe it's less confusing if you put it in an array? or maybe even make it a computed property

Reply:
I'm thinking due to the hierarchy needed for this it's probably better to do with custom CSS, utility classes don't work too great for this level of complexity and are just hard to read. Additionally we are trying to not use utility classes at all anymore in library code, going forward they should only be used at the application level because they are immutable.

```css
.emoji-row__reaction {
  height: var(--dt-size-550);
  border-radius: var(--dt-size-radius-500);
  ..... and so on (include values in else of reaction.isSelected too)

  &--selected {
    color: var(--dt-color-purple-500);
    background-color: hsla(var(--dt-color-purple400-hsl) / var(--dt-opacity-200));
    &:hover {
      ... on hover stuff
    }
  }
}
```

Reply:
we should not use utility classes in component anymore.
It will be necessary to create a custom class as brad shows.